LabApiUtilities/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    id 'java-library'
+    id 'com.microsoft.identity.buildsystem' version '0.1.0'
```

Comment:
We should update the buildsystem pluging documentation to note that the plugin is being used in both Android and Java projects now.

Reply:
Sure, I will do that